extensions/smallrye-reactive-messaging-kafka/runtime/pom.xml

```diff
@@ -22,6 +22,10 @@
             <groupId>io.smallrye.reactive</groupId>
             <artifactId>smallrye-reactive-messaging-kafka</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
```

Comment:
`quarkus-vertx` has dependency on `jackson-databind`

`quarkus-smallrye-reactive-messaging` depends on `quarkus-vertx`

`quarkus-smallrye-reactive-messaging-kafka` depends on `quarkus-smallrye-reactive-messaging`

Why is explicit `jackson-databind` dependency needed in `quarkus-smallrye-reactive-messaging-kafka`?

Reply:
I tried kafka-quickstart and native build failed without jackson-databind explicit dependency here.

But why is it needed  :( ?

Reply:
JVM mode doesn't need jackson-core, but native mode needs that in analysis phase.